sample-dcrdata.conf

```diff
@@ -3,12 +3,14 @@
 debuglevel=debug
 ;debuglevel=DATD=debug,DSQL=debug,MEMP=debug,DCRD=info,RPCC=info,JAPI=debug
 
-dcrduser=duser
-dcrdpass=asdfExample
+dcrduser=user
+dcrdpass=pass
 
-dcrdserv=localhost:9109
+dcrdserv=localhost:19109
 ;dcrdcert=/home/me/.dcrd/rpc.cert
-nodaemontls=0
+nodaemontls=1
```

Comment:
I am unable to get this PR to function with the original nodaemontls=0 setting. 0 should remain the default.